app/vmalert/group.go

```diff
@@ -111,6 +121,11 @@ func newGroup(cfg config.Group, qb datasource.QuerierBuilder, defaultInterval ti
 	if g.Interval == 0 {
 		g.Interval = defaultInterval
 	}
+
+	if g.Interval < g.IntervalOffset {
```

Comment:
I suggest to move this check for validate method of Group struct. 
```go
func (g *Group) Validate
```

Reply:
Did you mean to move it to `config` package, right?